book/docs/hello/notebooks/Lasso.ipynb

```diff
@@ -5,30 +5,38 @@
    "source": [
     "# LASSO\n",
     "\n",
+    "We wish to recover a sparse vector $x \\in \\mathbf{R}^n$\n",
+    "from measurements $y \\in \\mathbf{R}^m$.\n",
+    "Our measurement model tells us that\n",
     "\n",
-    "We wish to recover a sparse vector $x \\in \\mathbf{R}^n$ from measurements $y \\in \\mathbf{R}^m$. Our measurement model tells us that\n",
     "$$\n",
     "y = Ax + v,\n",
     "$$\n",
-    "where $A \\in \\mathbf{R}^{m \\times n}$ is a known matrix and $v \\in \\mathbf{R}^m$ is unknown measurement error.\n",
-    "For our demonstration the entries of $v$ are sampled from the normal distribution with mean zero and\n",
+    "\n",
+    "where $A \\in \\mathbf{R}^{m \\times n}$ is a known matrix\n",
+    "and $v \\in \\mathbf{R}^m$ is an unknown measurement error.\n",
+    "For our demonstration the entries of $v$ are sampled\n",
+    "from the normal distribution with mean zero and\n",
     "standard deviation $\\sigma$ (by default, $\\sigma = 1$).\n",
     "\n",
     "We can first try to recover $x$ by solving the optimization problem\n",
     "\n",
-    "\\begin{array}{ll} \\mbox{minimize} & ||Ax - y||^2_2 + \\gamma ||x||^2_2.\\\\\n",
-    "\\end{array}\n",
+    "$$\n",
+    "\\mbox{minimize} ||Ax - y||^2_2 + \\gamma ||x||^2_2.\n",
+    "$$\n",
     "\n",
     "This problem is called ridge regression.\n",
     "\n",
-    "The code below defines $n$, $m$, $A$, $x$, and $y$. Use CVXPY to estimate $x$ from $y$ using ridge regression. Try multiple\n",
+    "The code below defines $n$, $m$, $A$, $x$, and $y$.\n",
+    "Use CVXPY to estimate $x$ from $y$ using ridge regression.\n",
+    "Try multiple\n",
     "values of $\\gamma$. Use the plotting code to compare the estimated $x$ with the true $x$.\n",
     "\n",
     "A more effective approach is to solve the LASSO problem\n",
     "\n",
-    "\\begin{array}{ll} \\mbox{minimize} & ||Ax - y||^2_2 + \\gamma \\|x\\|_1.\\\\\n",
-    "\\end{array}\n",
-    "\n",
+    "$$\n",
```

Comment:
Stephen's styleguide uses \begin{array} and \end{array} for optimization problems